src/main/java/com/passpnu/passwordmanager/generator/StringPasswordGenerator.java

```diff
@@ -29,11 +28,8 @@ public String generate(){
         CharacterRule digitRule = new CharacterRule(EnglishCharacterData.Digit);
         digitRule.setNumberOfCharacters(digitCount);
 
-        CharacterRule specialCharRule = new CharacterRule(EnglishCharacterData.Special);
-        specialCharRule.setNumberOfCharacters(specialCharCount);
```

Comment:
I'm pretty sure adding special characters will be more secure. Why are they deleted?

Reply:
Cause some characters are too special... For example: †1A91Ldr)U, 4Yq6F06i¸³M‗T, -n1VTeVx2, 'r₭fPQ6bZ5H